COHHIO/ryha
# Improve documentation and tests

# Changes
- Added documentation (using roxygen comments) to some functions
- Added unit tests for some functions
- Added a new parameter to `check_file_names()` to be able to properly create the unit test
- Deleted tests for a non-existent function
- Applied some code styling

# Additional comment
- Functions that require a database connection (such as `read_data_from_table()` are not currently tested. We could create an in-memory database with `dbConnect(RSQLite::SQLite(), ":memory:")` but that would involve adding an extra dependency we don't want to add.

# Test
- Checkout to the branch and run devtools::test(). Some functions create temp folders and files to be able to perform tests. We need to make sure that this is implemented correcly
- Check that the new parameter of `check_file_names()` isn't breaking functionality. I would assume that it shouldn't
